instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/tracer/utils/NetPeerUtils.java

```diff
@@ -26,24 +26,7 @@
 
   private NetPeerUtils() {}
 
-  public static void setNetPeer(Span span, InetSocketAddress remoteConnection) {
-    if (remoteConnection != null) {
-      InetAddress remoteAddress = remoteConnection.getAddress();
-      if (remoteAddress != null) {
-        setNetPeer(span, remoteAddress);
-      } else {
-        // Failed DNS lookup, the host string is the name.
-        setNetPeer(span, remoteConnection.getHostString(), null);
-      }
-      span.setAttribute(SemanticAttributes.NET_PEER_PORT.key(), remoteConnection.getPort());
-    }
-  }
-
-  public static void setNetPeer(Span span, InetAddress remoteAddress) {
-    setNetPeer(span, remoteAddress.getHostName(), remoteAddress.getHostAddress());
-  }
-
-  public static void setNetPeer(Span span, String peerName, String peerIp) {
+  public static void setNetPeer(Span span, String peerName, String peerIp, int port) {
```

Comment:
Please review the order of method. Before this change it was somewhat logical if you look at parameter types. Now it seems to me as more chaotic.

Reply:
Goal of my refactoring was to make it in order - main method (setting attrs) first, then delegates starting from the simplest one. But I can bring it back as it was, just confirm that you'd like it that way :)

Reply:
I prefer the same order as with constructors: shorter before, longer after. If A delegates to B, then A is before B.

Reply:
Changed as suggested :)